Support the Column model for specifying the FK for an enum

ttung opened the pull request:
### Description
It is code-wise saner to specify the FK column with the python object rather than a string name.  When that is done, colspec is a `schemaname.tablename.columnname`.  In that case, we want the second-to-last split.

### Test plan
Used in subsequent PR.

phoenixAja commented on third-party/sqlalchemy-enum-tables/enumtables/alembic_autogen.py:
```diff
@@ -9,8 +9,10 @@
-		return tablename
+		splitted = colspec.split(".")
+		assert len(splitted) >= 2
+		# second to the last is the table name.
+		return splitted[-2]
```
i think remove comment and do this instead
```suggestion
		tablename = splitted[-2]
		return tablename
```

ttung replied:
I have updated this, incorporating some of your suggestion.  I will land this now, but if you still have concerns, I'm happy to revise.

phoenixAja commented on third-party/sqlalchemy-enum-tables/enumtables/alembic_autogen.py:
```diff
@@ -9,8 +9,10 @@
-		tablename, _ = colspec.split(".")
-		return tablename
+		splitted = colspec.split(".")
+		assert len(splitted) >= 2
```
will the length ever be > 2?

ttung replied:
Yes, it can be `schemaname`.`tablename`.`columnname`.

phoenixAja reviewed (approved):
small questions/comments

Commit:
Support the Column model for specifying the FK for an enum

### Description
It is code-wise saner to specify the FK column with the python object rather than a string name.  When that is done, colspec is a `schemaname.tablename.columnname`.  In that case, we want the second-to-last split.

### Test plan
Used in subsequent PR.
